src/field/gf_101_2.rs

```diff
@@ -322,7 +340,7 @@ mod tests {
 
   #[test]
   fn test_generator_order() {
-    let generator = F2::generator();
+    let generator = F2::ONE;
     let mut x = generator;
     for _ in 1..F2::ORDER {
       x *= generator;
```

Comment:
This isn't correct right? We are just multiplying one together a bunch of times, this test would pass no matter what right?

Reply:
oops lol yes, how did i miss this. fixed it now. Now, we're multiplying 1 by generator, order-1 times, which is the multiplicative order of the group defined by field.